projects/schematics/README.md

```diff
@@ -31,51 +29,32 @@ The following points provide guidance on how to achieve that.
 
 ### Integration testing
 
-The best way to test an unpublished schematic is to publish it to a local npm registry. For more, see [developing update schematics](#Developing-update-schematics)
-
-## Developing update schematics
-
-### Verdaccio setup
-
-To setup a local npm registry, we're going to use [verdaccio](https://github.com/verdaccio/verdaccio). To set it up, do the following:
-
-- install it: `npm install --global verdaccio`
-- run it in a new terminal tab / window: `verdaccio`
-- create an npm user: `npm adduser --registry http://localhost:4873`. This is only needed when setting up _verdaccio_ for the first time.
-
-### Using verdaccio
-
-Create a new angular project:
+The best way to test an unpublished schematic is to publish it to a local npm registry. For more, see [developing schematics](#Developing-schematics)
 
-- `ng new spartacus-schematics-test` and `cd spartacus-schematics-test`
-- add Spartacus by running e.g. `ng add @spartacus/schematics@<version> --baseUrl https://spartacus-demo.eastus.cloudapp.azure.com:8443/ --baseSite electronics-spa`. Note the `<version>` after `ng add @spartacus/schematics`. This should be lower than the one you're going to publish. E.g. if developing schematics for Spartacus 3.0, then you should install Spartacus 2.0.
-- create `.npmrc` in the root of the project and paste the following content to it: `@spartacus:registry=http://localhost:4873` to point to the local npm server only for the `@spartacus` scoped packages. From this moment on, `@spartacus` scoped packages will use the local npm registry.
-- commit the changes, if any.
+## Developing schematics
 
-You can now run any Spartacus schematics related command, e.g. `ng add @spartacus/schematics` or `ng update @spartacus/schematics`, and angular will pull the Spartacus schematics lib from _verdaccio_ instead from the public _npm_ registry.
+### Preparing setup
 
-The next step is to publish libraries to _verdaccio_.
+- Install verdaccio `npm i -g verdaccio` (only for the first time)
+- Create new angular project `ng new schematics-test --style=scss`
+- Run verdaccio script `ts-node ./tools/schematics/testing.ts`
 
 ### Publishing to verdaccio
 
-The simplest way to publish Spartacus libraries to _verdaccio_ is to use `scripts/publish-schematics-verdaccio.sh` script.
-
-> Before running the script, make sure _verdaccio_ is running: `$ verdaccio`.
-
-To use it, just run `./publish-schematics-verdaccio.sh`. This will build _all_ the relevant spartacus libs and publish them to _verdaccio_.
+- select option `publish` from the verdaccio script (it will bump package patch version and publish to verdaccio)
```

Comment:
It would be nice to explain that initially we should "build all libs", after which we don't have to do it (unless something changes that needs to be published of course).

Reply:
Also, do you think we can add an option to build each of the schematics separately?
This will be useful when working on a specific schematic, and only e.g. organization schematic should be built and published.

Reply:
I'll add those options as they will be needed.